src/librustc_llvm/ffi.rs

```diff
@@ -426,6 +426,14 @@ pub type OperandBundleDefRef = *mut OperandBundleDef_opaque;
 pub type DiagnosticHandler = unsafe extern "C" fn(DiagnosticInfoRef, *mut c_void);
 pub type InlineAsmDiagHandler = unsafe extern "C" fn(SMDiagnosticRef, *const c_void, c_uint);
 
+#[repr(C)]
```

Comment:
Please add a comment to mark where the enum came from, or preferably write an `LLVMRust` wrapper.